connector/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -2552,6 +2552,7 @@ class SparkConnectPlanner(
     // To avoid explicit handling of the result on the client, we build the expected input
     // of the relation on the server. The client has to simply forward the result.
     val result = SqlCommandResult.newBuilder()
+    val metrics = ExecutePlanResponse.Metrics.newBuilder()
```

Comment:
Add a comment here that this empty by default and filled only for commands below.